tools/cleaner.ts

```diff
@@ -32,9 +35,14 @@ async function main() {
       console.error(`Cannot save followup: ${followupAnonymized.id}`)
       console.trace(err)
     }
+    console.log(["Terminé", "Followup", followup_count].join(";"))
```

Comment:
Il faut ressortir le `console.log` du for sinon ça va print à tout les saves et pas seulement pour le nombre total à la fin